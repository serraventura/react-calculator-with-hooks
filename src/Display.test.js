describe("<Display />", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Display text={[]} />);
    expect(wrapper).toBeTruthy();
  });

  it("should display right expression with right result", () => {});

  it("should match snapshot", () => {});
});
