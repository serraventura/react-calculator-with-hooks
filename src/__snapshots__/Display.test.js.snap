// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display /> should match snapshot 1`] = `ShallowWrapper {}`;
